admin/scss/_actionTabs.scss

```diff
@@ -170,7 +170,7 @@ $border: 1px solid darken(#D9D9D9, 15%);
 					>li{
 						width: auto;
 						a.tab-nav-link{
-							@include box-sizing(border-box);
+							box-sizin: border-box;
 							@include duration(0.5s);
```

Comment:
Yes! [autoprefixer is current used with bootstrap](https://github.com/silverstripe/silverstripe-framework/pull/5027/files#diff-b9e12334e9eafd8341a6107dd98510c9R236) in the gulpfile and I see no reason why we can't use it to handle our own css as well. I'll get rid of the transition and duration mixins now which [don't actually need to be prefixed anymore](http://caniuse.com/#feat=css-transitions).